src/utils_format_kairosdb.c

```diff
@@ -181,7 +181,8 @@ static int values_to_kairosdb(char *buffer, size_t buffer_size, /* {{{ */
 
 static int value_list_to_kairosdb(char *buffer, size_t buffer_size, /* {{{ */
                                   const data_set_t *ds, const value_list_t *vl,
-                                  int store_rates) {
+                                  int store_rates, char * const *http_attrs,
```

Comment:
Since you're not modifying the attributes in this function this should be a pointer to a list of const char items.